cmd/core/cmd_ctx.go

```diff
@@ -3,17 +3,20 @@ package cmdcore
 import (
 	"io"
 
+	"github.com/flyteorg/flytectl/pkg/ext"
 	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/service"
 )
 
 type CommandContext struct {
-	adminClient service.AdminServiceClient
-	in          io.Reader
-	out         io.Writer
+	adminClient           service.AdminServiceClient
```

Comment:
remove this one?

Reply:
Will take this in my matchable entity pr . As changing this now will discard your approval